promptsource/utils.py

```diff
@@ -49,7 +51,20 @@ def get_dataset(path, conf=None):
         builder_instance.download_and_prepare()
         return builder_instance.as_dataset()
     else:
-        return datasets.load_dataset(path, conf)
+        return load_dataset(path, conf)
+
+
+def load_dataset(dataset_name, subset_name):
+    try:
+        return datasets.load_dataset(dataset_name, subset_name)
+    except datasets.builder.ManualDownloadError:
+        return datasets.load_dataset(
+            dataset_name,
+            subset_name,
+            data_dir=os.environ["PROMPTSOURCE_MANUAL_DATASET_DIR"]
```

Comment:
nit: it would make sense to me to have subfolders for each dataset? i.e. right now stuff (csv files for instance) for jigsaw will be under the same folder (`PROMPTSOURCE_MANUAL_DATASET_DIR`) as the stuff for story_cloze.
It could lead to file conflicts if everything is named "train.csv" or "test.csv"... thoughts?